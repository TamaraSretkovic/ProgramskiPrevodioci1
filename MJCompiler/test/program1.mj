//	Oporavak od greske deklaracija globalnih promenljivih 

program test302
    int aa, 123;
{
	void main() {
		
	}
}
