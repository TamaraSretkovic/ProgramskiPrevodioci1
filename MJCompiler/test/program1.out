INFO  01:15:53,276 - Compiling source file: /home/tamarica/Documents/tamara/velika skola/8 semestar/PP1/projekat/projecat_2019_2020/MJCompiler/test/program1.mj
INFO  01:15:53,299 - #2 program
INFO  01:15:53,300 - #46 test302
INFO  01:15:53,300 - #46 int
INFO  01:15:53,300 - #46 aa
INFO  01:15:53,301 - #6 ,
INFO  01:15:53,301 - #47 123
ERROR 01:15:53,301 - 
Sintaksna greska na liniji 4
INFO  01:15:53,301 - #5 ;
INFO  01:15:53,301 - #3 {
INFO  01:15:53,301 - #10 void
INFO  01:15:53,302 - #46 main
ERROR 01:15:53,302 - Uspesan oporavak od greske na liniji 4 pri deklarisanju varijable do , ili ;
INFO  01:15:53,303 - #12 (
INFO  01:15:53,303 - #11 )
INFO  01:15:53,304 - #3 {
INFO  01:15:53,304 - #4 }
INFO  01:15:53,304 - #4 }
ERROR 01:15:53,305 - Parsiranje NIJE uspesno zavrseno!
